cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -0,0 +1,288 @@
+
+// TODO: Move the theme variables to cf-enhancements.
+/* topdoc
+  name: Theme variables
+  family: cf-core
+  notes:
+    - "The following color and sizing variables are exposed, allowing you to
+       easily override them before compiling."
+  patterns:
+    - name: Sizing
+      codenotes:
+        - |
+          @margin__em
+          @margin_half__em
+  tags:
+    - cf-core
+*/
+
+@margin__em: unit( @grid_gutter-width / @base-font-size-px, em );
+@margin_half__em: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
```

Comment:
Even if it's em, if the font-size of the elem it's used in isn't 16px, the computed value will change from the expected 30 or 15px. ex.

```
.thing {
  font-size: unit( 18px / @base-font-size, em );
  margin-bottom: @base-space__em; // This will be computed as 1.875em * 18px = 33.75px
}

.parent-thing {
  @heading-5();

  .child-thing {
    margin-bottom: @base-space_half__em; // This will be computed as 0.9375em * 14px = 13.125px
}
```